src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -291,14 +291,78 @@ static void processGenericEntities
 
 /* ****************************************************************************
 *
-* mongoQueryContext - 
+* cerVectorPresent -
```

Comment:
This is temporal code for debugging? Please mark it with a "FIXME to be removed after debugging ends" or something like that.

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed